rero/rero-ils-ui
# patron_types: update detail view with new fields

New fields are added to patron_types resource in the backend. Updates the detail view
to reflect theses changes.

linked to rero/rero-ils#865

Co-Authored-by: Renaud Michotte <renaud.michotte@gmail.com>

## Why are you opening this PR?

https://tree.taiga.io/project/rero21-reroils/us/1297?milestone=257446

## How to test?

* Use `US1297` branch from `rero/rero_ils` 
* Create/update a patron type to set a subscription amount.
* display the detail of the edited patron type. The display should be like images below

With yearly subscription
![image](https://user-images.githubusercontent.com/10031585/77079010-aab44180-69f7-11ea-9044-4c0cc6267d3b.png)

Whitout yearly subscription
![image](https://user-images.githubusercontent.com/10031585/77079095-c4ee1f80-69f7-11ea-9816-280e2e219d8b.png)



## Code review check list
